packages/graph-explorer/src/modules/ConnectionDetail/ConnectionDetail.tsx

```diff
@@ -217,6 +217,7 @@ const ConnectionDetail = ({ isSync, onSyncChange }: ConnectionDetailProps) => {
             url: config.connection?.url,
             type: config.connection?.queryEngine,
             fetchTimeMs: config.connection?.fetchTimeoutMs,
+            serviceType: config.connection?.serviceType,
```

Comment:
If the connection is optional, could the value be empty?

Reply:
yes, there is validation + default value for such cases